pkgs/by-name/re/rerun/package.nix

```diff
@@ -31,7 +32,17 @@
     "map_view"
   ],
 }:
+let
+  clang-unwrapped = llvmPackages.clang-unwrapped;
+  libclang = llvmPackages.libclang.lib;
+  bintools-unwrapped = llvmPackages.bintools-unwrapped;
 
+  major-version = builtins.head (builtins.splitVersion clang-unwrapped.version);
+
+  # resource dir + builtins from the unwrapped clang
+  resourceDir = "${clang-unwrapped.lib}/lib/clang/${major-version}";
+  includeDir = "${libclang}/lib/clang/${major-version}/include";
```

Comment:
This could go in a `let ... in` statement in the `env` below, making it more local.